src/SuperstateToken.sol

```diff
@@ -97,10 +97,9 @@ abstract contract SuperstateToken is ERC20Upgradeable, IERC7246, PausableUpgrade
      * @param _allowList Address of the AllowList contract to use for permission checking
      * @dev Disables initialization on the implementation contract
      */
-    constructor(address _admin, AllowList _allowList) {
+    constructor(address _admin, AllowList _allowList)  {
```

Comment:
```suggestion
    constructor(address _admin, AllowList _allowList) {
```
pls run `forge fmt`

Reply:
Addressed in
- https://github.com/superstateinc/ustb/pull/56/commits/4e52d672532d51249bce9677ae1801a91404eda8
- https://github.com/superstateinc/ustb/pull/56/commits/863dda0005df872dec9136d026df07f918dbb2e5